### ISSUE TITLE
Support gifs

### ISSUE THREAD

**Issue by francoislg** [francoislg/PPaaS#2]
Allows to import animated gifs and merge them

**Comment by gustavohenke** [comment]
Yes, please! Lovely :southamericanparrot: needed in our Slack 😁 ![](https://emoji.slack-edge.com/T028TG1VC/southamerican/08bc71873d6995db.gif)
